CrewChiefV4/App.config

```diff
@@ -356,14 +353,26 @@
       <setting name="rf1_spotter_car_length" serializeAs="String">
         <value>4.5</value>
       </setting>
-      <setting name="rf1_launch_params" serializeAs="String">
-        <value />
+      <setting name="rf1_steam_id" serializeAs="String">
+        <value>-applaunch 339790</value>
       </setting>
-      <setting name="rf1_launch_exe" serializeAs="String">
-        <value />
+      <setting name="steam_exe" serializeAs="String">
+        <value>C:/Program Files (x86)/Steam/Steam.exe</value>
       </setting>
       <setting name="launch_rfactor1" serializeAs="String">
-        <value>False</value>
+        <value>True</value>
```

Comment:
In the forums it appears that the average user is expecting the default behavior to launch the game automatically without further configuration. I agree though that as a rule, 'False' is a much safer default. I'll change it back to 'False' and make a note in the installation procedures on the forums.